ush/python_utils/config_parser.py

```diff
@@ -465,25 +465,26 @@ def update_dict(dict_o, dict_t, provide_default=False):
 
 def check_structure_dict(dict_o, dict_t):
     """Check if a dictionary's structure follows a template.
-    The invalid entries are printed to the screen.
+    The invalid entries are returned as a list of lists.
+    If all entries are valid, returns an empty dictionary
 
     Args:
         dict_o: target dictionary
         dict_t: template dictionary to compare structure to
     Returns:
-        Boolean
+        inval:  dictionary of invalid key-value pairs
```

Comment:
```suggestion
        dict:  Invalid key-value pairs.
```
In this docstring style, left of the colon should be the return type.